docroot/profiles/custom/cgov_site/themes/custom/cgov/webpack.base.config.js

```diff
@@ -30,7 +31,10 @@ const config = {
 					{
 						loader: MiniCssExtractPlugin.loader
 					},
-					'css-loader',
```

Comment:
this block can be reverted as it doesn't do anything.  The minify option was deprecated after v1.0.0

Reply:
Done!